roles/update_status_html/tasks/main.yml

```diff
@@ -34,6 +34,24 @@
     ansible_docker_migrid_version: "{{ { 'ansible_docker_migrid_version' : result.stdout } }}"
   when: result.rc==0
 
+- name: register dockerfile version
+  set_fact:
+    ansible_dockerfile_version: "{{ { 'dockerfile_version' : migrid_container_type } }}"
```

Comment:
I would write this either so that it reports the full Dockerfile name

```
    migrid_dockerfile_version: "{{ { 'dockerfile_version' : "Dockerfile." + migrid_container_type } }}"
```

or that it just reports the `migrid_container_type`:

```
    migrid_used_contanier_tyoe: "{{ { 'migrid_container_type' : migrid_container_type } }}"
```

The variable name ansible_dockerfile_version is a bit confusing to me.
I would prefer migrid_dockerfile_version

Reply:
now changed to migrid_dockerfile_version